code/src/components/form.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Form = ({ newThought, setNewThought, formSubmit }) => {
+return (
+    <div className="form-wrapper">
+    <form onSubmit={formSubmit}>
+        <label htmlFor="newThought">What is making you happy right now?</label>
+        <div>
+        <textarea
+            id="newThought"
+            type="text"
+            rows="5"
+            value={newThought}
+            onChange={(e) => setNewThought(e.target.value)}
+            placeholder="Happy thought here!"
+        />
+        <p className="letter-counter">
```

Comment:
Nice! It looks so simple, I was wondering how you did that when I look att your Netlify...